Sources/Conformance/main.swift

```diff
@@ -120,6 +120,8 @@ func singleTest() throws -> Bool {
 }
 
 Google_Protobuf_Any.register(messageType: Conformance_TestAllTypes.self)
+Google_Protobuf_Any.register(messageType: ProtobufTestMessages_Proto3_TestAllTypes.self)
```

Comment:
Startup - The catch is someone could be using code that uses protos internally, so they don't know what it has to have registered.  Forcing those modules to add a _Startup_ method that has to be called also sorta sucks.

I guess I should go look at what you have on Any (I haven't yet).  The other languages all don't do any implicit decoding of an Any.  Instead when the outer message is decoded, the Any stays with it's content as a blob and there are helpers to extract/decode the Any to specific types.  An application doesn't anyways want the Any decoded, and even when it does, the code is only going to know how to deal with a specific number set of them.